how_to_qa/README.md

```diff
@@ -0,0 +1,40 @@
+class: center middle
+# Test Tooling as a service
+
+QA should be about preventing defects(!bugs), Not finding them.
+
+---
+## Ideal Workflow
+- For every new epic, SET determines if we have sufficient tooling to test the new feature.
+
+- SET build/enhance existing automation tools and test frameworks as per need.
+
+- Non-feature Dev utilizes these tools and framework to write feature tests (non-unit tests).
+
+- SET reviews the test coverage, and sign off the release.
+
+---
+## Role of SET
+- SETs should be quality advocates, influencing both development and operational processes.
+
+- SET focus on how to improve our quality stack, and make it efficient.
+
+    - Build and maintain tools to improve operational ease.
+
+    - Tooling to support continuous integration/continuous deployment model.
+
+    - Tools for contracts testing, resiliency testing, load testing and few more.
```

Comment:
If i think more, SET writing this makes more sense, since that part is outside the new feature dependencies
and can be done in isolation.
Same argument can be made for Load testing.